libs/SalesforceHybridSDK/SalesforceHybridSDK/Classes/Plugins/SFSmartStore/SFSmartStorePlugin.m

```diff
@@ -51,63 +52,48 @@
 NSString * const kPathsArg            = @"paths";
 NSString * const kReIndexDataArg      = @"reIndexData";
 NSString * const kIsGlobalStoreArg    = @"isGlobalStore";
+NSString * const kStoreName           = @"storeName";
 
 @interface SFSmartStorePlugin() 
 
-@property (nonatomic, strong) SFSmartStoreInspectorViewController *inspector;
-@property (nonatomic, strong) SFSmartStoreInspectorViewController *globalInspector;
-@property (nonatomic, strong) SFSmartStore *store;
-@property (nonatomic, strong) SFSmartStore *globalStore;
-@property (nonatomic, strong) NSMutableDictionary *userCursorCache;
-@property (nonatomic, strong) NSMutableDictionary *globalCursorCache;
-
+@property (nonatomic, strong) SFMutableDictionaryThreadSafe *databaseInspectors;
```

Comment:
This is the continuation of an existing anti-pattern that I'd prefer to see squashed.  This idea of persisted inspector view controllers is unnecessary—the persistence of the view controller should be managed by the view state.  I.e. `self.viewController` presents it, it exists.  `self.viewController` dismisses it, it's reclaimed.  And the rest of the state of it should exist as properties inside the view controller class itself.

Can we clean this up?